homeassistant/components/wirelesstag/binary_sensor.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, api, tag, sensor_type):
         super().__init__(api, tag)
         self._sensor_type = sensor_type
         self._name = f"{self._tag.name} {self.event.human_readable_name}"
+        self._attr_unique_id = f"{WIRELESSTAG_DOMAIN}_{self.tag_id}_{self._sensor_type}"
```

Comment:
No need to include the domain, unique IDs only have to be unique within an integration+platform.

Reply:
```suggestion
        self._attr_unique_id = f"{self.tag_id}_{self._sensor_type}"
```